### PR TITLE
MODSOURMAN-621 Fix saving of default mapping rules

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
Fix saving of default mapping rules

## Approach
Do not override mapping rules if they are already exist in database

### CODE REVIEW

File: mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -168,6 +168,11 @@
       "snippet": "DO $$ BEGIN BEGIN create type rule_type AS ENUM ('MARC_BIB', 'MARC_HOLDING'); EXCEPTION WHEN duplicate_object THEN NULL; END; END $$; ALTER TABLE mapping_rules ADD COLUMN IF NOT EXISTS record_type rule_type NOT NULL DEFAULT 'MARC_BIB';",
       "fromModuleVersion": "mod-source-record-manager-3.2.0"
     },
+    {
+      "run": "after",
+      "snippet": "ALTER TABLE mapping_rules ADD CONSTRAINT unq_record_type UNIQUE (record_type);",
+      "fromModuleVersion": "mod-source-record-manager-3.2.6"
```
Comment (psmagin) [inline mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json:174]: I'm not very familiar with schema updating in RMB. Can you approve that I set the right version?
This change will be cherry-picked to kiwi release branch

Comment (RuslanLavrov) [reply mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json:174]: hi @psmagin, yes the fromModuleVersion is correct

**Review by oleksandr-dekin — APPROVED** [review]

File: mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -168,6 +168,11 @@
       "snippet": "DO $$ BEGIN BEGIN create type rule_type AS ENUM ('MARC_BIB', 'MARC_HOLDING'); EXCEPTION WHEN duplicate_object THEN NULL; END; END $$; ALTER TABLE mapping_rules ADD COLUMN IF NOT EXISTS record_type rule_type NOT NULL DEFAULT 'MARC_BIB';",
       "fromModuleVersion": "mod-source-record-manager-3.2.0"
     },
+    {
+      "run": "after",
+      "snippet": "ALTER TABLE mapping_rules ADD CONSTRAINT unq_record_type UNIQUE (record_type);",
```
Comment (KaterynaSenchenko) [inline mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json:173]: Hi @psmagin, as we discussed with Khalilah and Ann-Marie, let's remove the constraint on the db for now

Comment (psmagin) [reply mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json]: Done

**Review by RuslanLavrov — APPROVED** [review]

**Pull request merged**